src/chat.rs

```diff
@@ -5679,6 +5680,51 @@ mod tests {
         .await
     }
 
+    #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+    async fn test_sticker_jpeg_force() {
+        let alice = TestContext::new_alice().await;
+        let bob = TestContext::new_bob().await;
+        let alice_chat = alice.create_chat(&bob).await;
+
+        let file = alice.get_blobdir().join("sticker.jpg");
+        tokio::fs::write(
+            &file,
+            include_bytes!("../test-data/image/avatar1000x1000.jpg"),
+        )
+        .await
+        .unwrap();
+
+        // Images without force_sticker should be turned into [Viewtype::Image]
```

Comment:
Afair, there's already the test on this, `test_recode_image_balanced_png()`

Reply:
Yes, I copied that part. Made that so it is more obvious what actually happens with `force_sticker`

Reply:
Then it can be removed from `test_recode_image_balanced_png()`. This test is a better place for this check, indeed

Reply:
the tests in `test_recode_image_balanced_png()` checks for resolution, recording and exif. this is a different test from the test here.

i would re-add the sticker-test to `test_recode_image_balanced_png()` - even if there is some overlap, this is fine

Reply:
same